samples/language/EntryPoint.qs

```diff
@@ -0,0 +1,17 @@
+/// # Sample
+/// EntryPoint
+///
+/// # Description
+/// When defining an entry point into a Q# program, the Q# compiler recognizes
+/// the @EntryPoint() attribute rather than requiring that entry points have a
+/// particular name, for example, main, Main, or __main__. That is, from the
+/// perspective of a Q# developer, entry points are ordinary operations annotated
+/// with @EntryPoint().
+
+namespace MyQuantumApp {
+
+    @EntryPoint()
```

Comment:
```suggestion
    // The Q# compiler identifies `Main` as the entry point operation because it is marked with the `@EntryPoint()` attribute.
    @EntryPoint()
```

Reply:
Including additional comments in the code sections that are relevant to the sample topic would be helpful for AI training even if it seems a bit redundant.